src/components/flow/ResultsAsSimulation.tsx

```diff
@@ -1,24 +1,446 @@
+						Math.max(
+							0,
+							cars.get(exitEntrancePoint.index)! +
+								joinedCars.get(exitEntrancePoint.index)! -
+								leftCars.get(exitEntrancePoint.index)!,
+						),
+					);
+				}
+				setCars(newCars);
+			}, timeDelta);
+			return () => clearInterval(lightsTimer);
+		}
+	}, [running, timer]);
+
+	const ShowLight = (
+		usedLights: TrafficLight[],
+		lightsSeq: number[][],
+		roadFlow: number,
+		point: ExitEntrancePoint,
+	) => {
+		let idx = point.index * -1;
+		let lightColor = LightColors.BLACK;
+		const carNumber = cars.get(point.index)!;
+
+		const result = [];
+
+		for (let i = 0; i < usedLights.length; i++) {
+			const light = usedLights[i];
+			idx = light.index;
+			lightColor = lights.get(usedLights[i].index)!;
+
+			if (i == 0) {
+				result.push(
+					<div key={idx}>
+						<EEIPointMarker
+							key={idx}
+							color={lightColor}
+							yCord={point.yCord}
+							xCord={point.xCord}
+						>
+							<SimulationNumbers>{carNumber}</SimulationNumbers>
+						</EEIPointMarker>
+					</div>,
+				);
+			} else {
+				result.push(
+					<div key={idx}>
+						<EEIPointMarker
+							key={idx}
+							color={lightColor}
+							yCord={point.yCord}
+							xCord={point.xCord + 15 * i}
+						></EEIPointMarker>
+					</div>,
+				);
+			}
+		}
+
+		if (result.length == 0) {
+			result.push(
+				<div key={idx}>
+					<EEIPointMarker
+						key={idx}
+						color={lightColor}
+						yCord={point.yCord}
+						xCord={point.xCord}
+					>
+						<SimulationNumbers>{}</SimulationNumbers>
+					</EEIPointMarker>
+				</div>,
+			);
+		}
+		return result;
+	};
+
+	const handleChooseButton = (current_state: pauseButtonState) => {
+		console.log(current_state, running);
+		if (current_state === "paused") {
+			setRunning(false);
+		} else {
+			setRunning(true);
+		}
+	};
 
 	return (
-		<>
+		<ContainerDiv>
 			<Navbar />
-			<div>
-				<h3>Results as simulation</h3>
-				<NeutralNegativeButton>
-					<BaseButtonLink to="../results-choice" relative="path" state={all}>
-						Go back to results choice panel
-					</BaseButtonLink>
-				</NeutralNegativeButton>
-			</div>
-		</>
+			<PageHeader>{crossroadName}</PageHeader>
+			<BorderedWorkaroundDiv>
+				{connections.length > 0 &&
+					connections.map((con) => {
+						const entrancePoint = exitEntrancePoints.filter(
+							(point) => point.id === con.sourceId,
+						)[0];
+						const exitPoint = exitEntrancePoints.filter(
+							(point) => point.id === con.targetId,
+						)[0];
+
+						return (
+							<ConnectionMarker
+								key={con.index}
+								thickness={3}
+								entranceX={entrancePoint.xCord}
+								entranceY={entrancePoint.yCord}
+								exitX={exitPoint.xCord}
+								exitY={exitPoint.yCord}
+								connection={con}
+								color={Colors.BRIGHT_RED}
+								withLightIds={true}
+								withTooltip={false}
+							/>
+						);
+					})}
+				{exitEntrancePoints.length > 0 && (
+					<ThemeProvider theme={tooltipTheme}>
```

Comment:
To się pojawia w kilku miejscach, po co używasz ThemeProvidera do Tooltipa, jeśli nie używasz Tooltipa
ConnectionMarker ma go wbudowanego, a poza tym dajesz `withTooltip={false}`, a EEIPointMarkerów w ogóle nie owijasz Tooltipem?